data/operators/amenity/bicycle_parking.json

```diff
@@ -34,6 +34,21 @@
         "operator:wikidata": "Q112136998"
       }
     },
+    {
+      "displayName": "Verkehrsverbund Vorarlberg",
+      "locationSet": {
+        "include": ["at"]
+      },
+      "matchNames": ["cleVVVer", "vmobil"],
+      "tags": {
+        "amenity": "bicycle_parking",
+        "name": "Verkehrsverbund Vorarlberg",
+        "operator": "Verkehrsverbund Vorarlberg GmbH",
+        "operator:short": "VVV",
+        "operator:wikidata": "Q2516495",
+        "operator:wikipedia": "de:Verkehrsverbund_Vorarlberg"
```

Comment:
> @bhousel Is operator:wikipedia a best practice or not welcome here?

In #6481 all wikipedia tags removed, see more in https://github.com/osmlab/name-suggestion-index/wiki/Judge-Case